src/components/graph/Graph.jsx

```diff
@@ -290,7 +292,7 @@ export default class Graph extends React.Component {
 
     const zoomObject = d3Zoom()
       .scaleExtent([this.state.config.minZoom, this.state.config.maxZoom])
-      .on("zoom", this._zoomed);
+      .on("zoom", !this.state.config.freezeAllDragEvents ? this._zoomed : null);
```

Comment:
Maybe we don't need to trigger `.on("zoom"), ...` at all, consider the alternative:

```javascript
const zoomObject = d3Zoom()
      .scaleExtent([this.state.config.minZoom, this.state.config.maxZoom]);

if (!this.state.config.freezeAllDragEvents) {
  zoomObject.on("zoom", this._zoomed);
}
```

Reply:
Ah sure, I can make that change!